homeassistant/components/tuya/sensor.py

```diff
@@ -262,6 +262,29 @@ class TuyaSensorEntityDescription(SensorEntityDescription):
             translation_key="last_amount",
             state_class=SensorStateClass.MEASUREMENT,
         ),
+        TuyaSensorEntityDescription(
+            key=DPCode.BATTERY_PERCENTAGE,
+            translation_key="battery",
+            native_unit_of_measurement=PERCENTAGE,
+            device_class=SensorDeviceClass.BATTERY,
+            state_class=SensorStateClass.MEASUREMENT,
+            entity_category=EntityCategory.DIAGNOSTIC,
+        ),
+        TuyaSensorEntityDescription(
+            key=DPCode.MEAL_PLAN,
+            translation_key="meal_plan",
+            state_class=SensorStateClass.MEASUREMENT,
+        ),
+        TuyaSensorEntityDescription(
+            key=DPCode.FEED_REPORT,
+            translation_key="feed_report",
+            state_class=SensorStateClass.MEASUREMENT,
+        ),
```

Comment:
What would the content of this look like?

Reply:
{“range”:[“standby”,“feeding”,“done”]}